tools/common.props

```diff
@@ -14,7 +14,4 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Key.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
-  <ItemGroup>
```

Comment:
This shouldn't be removed. This pulls in a newer Roslyn compiler that keeps our usage of Span correct. If you are having issues when using this compiler, lets discuss how to fix it.